src/cms/models/regions/region.py

```diff
@@ -181,6 +181,12 @@ class Region(models.Model):
         ),
     )
 
+    short_url = models.BooleanField(
```

Comment:
```suggestion
    short_urls_enabled = models.BooleanField(
```
I think for consistency reasons, we should stick with the naming scheme `_enabled` for boolean fields (like `events_enabled`, `statistics_enabled`and `chat_enabled` etc)